server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss

```diff
@@ -89,6 +89,15 @@ See the radio-option below as an example.
   @include u-height(1);
 }
 
+.cf-footer-primary-link {
+  color: var(--Base-ink, #1B1B1B);
```

Comment:
**suggestion:** I think some of this could be replaced with `@include u-text('base-darkest')` and same for some values throughout this file.

Reply:
Good idea!